### DISCUSSION TITLE
no level addded

### DISCUSSION THREAD

**Discussion by Aasthaa10** [andoriyaprashant/DevDocsHub#148] [Q&A] [answered]
pr link: https://github.com/andoriyaprashant/DevDocsHub/issues/143#issuecomment-2452957904
assigned the issue but not added the level....

**Comment by andoriyaprashant** [comment, marked as answer]
> pr link: https://github.com/andoriyaprashant/DevDocsHub/issues/143#issuecomment-2452957904
> 
> assigned the issue but not added the level....

Hello @Aasthaa10 I add the level label after reviewing the pull request. You can submit your pull request, and I'll review it and then add the labels.

**Reply by Aasthaa10** [reply]
Okk
